migration/identity-migration/src/main/java/io/camunda/migration/identity/config/oidc/StaticEntities.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.config.oidc;
+
+import io.camunda.migration.identity.config.oidc.OidcProperties.Audience;
+import io.camunda.service.AuthorizationServices.CreateAuthorizationRequest;
+import io.camunda.zeebe.protocol.record.value.AuthorizationOwnerType;
+import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
+import io.camunda.zeebe.protocol.record.value.PermissionType;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class StaticEntities {
+
+  private static final String IDENTITY_RESOURCE_ID = "identity";
+  private static final String OPERATE_RESOURCE_ID = "operate";
+  private static final String TASKLIST_RESOURCE_ID = "tasklist";
+
+  public static List<CreateAuthorizationRequest> getAuthorizationsByAudience(
```

Comment:
🔧 we could actually make this a bean, pass in the properties on creation and then the RoleHandler doesn't need to pass along the audiences every time? we could also have the debug log statement in the method that created the bean or in the ctor.

Given that, the RoleHandler might actually be identical to the one for SM? 🤔 
We could then have that bean setup actually the same way for keycloak and oidc, as in case of keycloak we may just always pass in an `Audiences` object with default values?

Reply:
I'll merge together the 2 handlers, using the default properties for the keycloak setup